app/Services/Model/Imp/SummitService.php

```diff
@@ -1501,6 +1501,17 @@ public function addSummit(array $data)
                 }
             }
 
+            $registration_slug_prefix = $data['registration_slug_prefix'] ?? null;
+            if (!empty($registration_slug_prefix)) {
+                // check if exist another summit with that registration slug
+
+                $old_summit = $this->summit_repository->getByRegistrationSlugPrefix(trim($registration_slug_prefix));
+                if (!is_null($old_summit)) {
+                    throw new ValidationException(sprintf(
+                        "registration slug prefix %s already belongs to another summit", $registration_slug_prefix));
```

Comment:
fix the error message
should be
Registration slug prefix %s already belongs to another summit (%s).
where the second %s is the former summit id

Reply:
done